plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveFileMetastore.java

```diff
@@ -85,4 +87,64 @@ public void testInsertOverwriteUnpartitioned()
     {
         // FileHiveMetastore has various incompatibilities
     }
+
+    @Override
+    public void testUpdateBasicPartitionStatistics()
+            throws Exception
+    {
+        // updating partition statistics in FileMetastore use single partition statistics, evaluating 0 to NaN is not possible
```

Comment:
`evaluating 0 to NaN is not possible` -> `treating partitions with zero row count statistics as empty statistics`